PLAN.md

```diff
@@ -0,0 +1,20 @@
+- `activeEditor.mode` isn't working correctly
+- Add `neovim` capability to activeEditor, that can be queried
+
+- Add option to hide commands in screen
+
+- Create these 'commands' in `Commands.ts`:
```

Comment:
I will look into this, I have a few things I want to implement with the menu that isn't supported such as selecting multiple files (c-i i.e. tab, (increment)) and opening things silently (i.e. open a file but leave the menu where it's open (whatever + shift so, c-S to split horizontal silent)), and to use c-w to delete word back and c-u to delete entire line of input field (linux standards I love)

The bindings will hook up to quickOpen which will be nice, I'm really liking this commit!